addon/services/session.js

```diff
@@ -18,6 +18,8 @@ const { computed, A, Service, Evented, getOwner, isNone, assert }  = Ember;
   });
   ```
 
+  and is __automatically injected by Ember Simple Auth's mixins__.
```

Comment:
I wouldn't add this here as it might lead to confusion. In most apps, you'd only be including the mixins in 3 routes (`application`, sth. like `internal` and `login`) but would use the service in controllers etc. as well.